services/ui-src/src/page/section/DetailSection.tsx

```diff
@@ -67,25 +70,26 @@ export const DetailSection = ({
           </section>
           {userRoleObj.canAccessForms ? (
             <section className="package-actions">
-              <h2>{pageConfig.actionLabel}</h2>
-              <ul className="action-list">
-                {pageConfig.actionsByStatus[detail.currentStatus]?.length >
-                0 ? (
-                  pageConfig.actionsByStatus[detail.currentStatus]?.map(
-                    (actionName, i) => (
-                      <li key={i}>
-                        {actionComponent[actionName](detail, updateData)}
-                      </li>
+              <Review heading={pageConfig.actionLabel}>
```

Comment:
I feel like it makes sense to have the Status label be a Review heading and I know the styles are the same for the Package Actions label.... but.... I am concerned that "Package Actions" is more of a section header than an item header for screen reading purposes?  Anyway, I noticed it and thought it was worth a mention....